source/features/download-folder-button.tsx

```diff
@@ -39,7 +39,7 @@ void features.add({
 		pageDetect.isRepoTree
 	],
 	exclude: [
-		pageDetect.isRepoRoot // Already has an native download ZIP button
+		() => select.exists('get-repo-controller') // Already has an native download ZIP button
```

Comment:
Might be worth keeping them both
```suggestion
		pageDetect.isRepoRoot, // Already has an native download ZIP button
		() => select.exists('get-repo-controller') // Same as `isRepoRoot` but supports branches with slashes
```

Better yet:
- **add** the `exists()` to `isRepoRoot` in `github-url-detection` ("if no URL is supplied, also check the dom")
- look for a meta tag in the `head` instead; if it exists, this is generally safer (because it can be run even on `waitForDom: false`)

Reply:
🎉 `<meta name="selected-link" value="repo_source" data-pjax-transient>`

Reply:
@fregante $('meta[name="selected-link"]') is an awesome selector! It works on almost all pages I tested and it gives back what page your on!

Reply:
@fregante $('meta[name="selected-link"]') is an awesome selector! It works on almost all pages I tested and it gives back what page your on!

Reply:
You mean `[name="analytics-location"]`?

Reply:
We may have to use both

Reply:
Neither one seems to apply to this case though.

This is true on any page of the Code tab. It might be worth adding as a new `isRepoSource` or `isRepoCodeTab` which is probably useful in many cases.
```js
$('meta[name="selected-link"]').getAttribute('value') === 'repo_source'
```

Reply:
I've opened a PR to improve `isRepoRoot`: https://github.com/fregante/github-url-detection/pull/16